src/coreComponents/mesh/generators/CellBlockManager.cpp

```diff
@@ -53,196 +54,111 @@ ArrayOfArrays< localIndex > CellBlockManager::getNodeToElements() const
 
   // First step: how many elements for each node, stored in the elemsPerNode array.
   array1d< localIndex > elemsPerNode( m_numNodes );
-  RAJA::ReduceSum< parallelHostReduce, localIndex > totalNodeElems = 0;
+  elemsPerNode.setValues< serialPolicy >( elemMapExtraSpacePerNode() );
```

Comment:
This function `getNodeToElements()` seems pretty similar to the `buildToElementMaps`. Any way to combine the two?

Reply:
I would like to, but they produce different types of values (one `ArrayOfArrays` vs a struct of three of them). With enough effort, the differences could be abstracted through user-provided functions, but I'm not sure the result would be pretty. The more important question though is whether we want to keep `getNodeToElements()` at all as in my opinion it produces some non-intuitive results (see https://github.com/GEOSX/GEOSX/pull/1847#discussion_r837053548)